### DISCUSSION TITLE
404 error code not found

### DISCUSSION THREAD

**Discussion by Praveen03AJJARAPU** [vercel/vercel#10424] [Help]
### Question

After successfully running my code on localhost, I deployed it into the Vercel but it showed me directly the 404 error not found. I then added the vercel.json with routes in it after that it also caused a 404 error not found. And I have done it with Vite at first I didn't recognise I thought it was the problem but it was not. Here is the Vercel URL https://facebook-clone-ypnt.vercel.app and my repo link is https://github.com/Praveen03AJJARAPU/facebook-clone

**Comment by idilaydindogan** [comment]
hi there, have you managed to solve this problem? i having the same and can not solve so far.. thx.
